src/acl/acl.ts

```diff
@@ -622,7 +622,9 @@ export async function deleteAclFor<
 
   if (!response.ok) {
     throw new Error(
-      `Deleting the ACL failed: ${response.status} ${response.statusText}.`
+      `Deleting the ACL of the Resource at \`${getSourceUrl(
```

Comment:
Just to note - the convention we use throughout ESS is to delimit variables in messages with '[' and ']' - so I think it makes sense to align with that (i.e. rather than using backticks instead).

Reply:
I don't have a _strong_ preference, but given that the consumers of ESS error messages and solid-client error messages are going to be quite different people, and that the latter have a common culture of marking raw content with backticks, I do have a slight preference for aligning with our ecosystem if that's OK with you.

Reply:
Yeah, I don't have a strong preference either, the most important thing is simply consistency. Ultimately I'd actually like to see all these messages expressed in RDF (allowing us provide unique identifiers for each of them, allowing for translations, allowing meta-data like 'seeAlso', etc.), and not hard-coded like this at all, and so in fact I don't see particular audiences as being too important. I'll discuss with Jack and see what his take on it is (since whatever we decide should be in the coding guidelines to try and ensure we get that consistency).